iina/iina
# Creating plugins using iina-cli

**Description:**

- Added plugin-related commands to `iina-cli`.
  - `create`: See https://github.com/iina/iina-plugin-template for more description.
  - `pack`: Create a zip file with the plugin folder's content.
  - `link`: Create a symlink in IINA's plugin folder.
  - `unlink`: Remove the above symlink.
- Fixed a crash when installing local `.iinaplgz` plugins.

Recommended steps for testing:

- Make sure that `iina-cli` can create plugins using various options.
- If building is required for the generated plugin (`iina-cli` will print instructions), make sure `npm install` and `npm run build` finish successfully.
- Make sure that `link` and `unlink` work for the generated plugin folder, i.e. IINA can load the folder as a plugin correctly.
- Make sure IINA can load the zip archive generated by `pack`.
